src/pages/orders/OrderDetails.js

```diff
@@ -86,56 +102,93 @@ const OrderDetails = (props) => {
             <hr />
             {/* Order Details */}
             <h2>Order Details</h2>
-            <div>
-              <div className="d-flex">
+            {console.log(dateTime, dateTime.getTime())}
+            <div className="pl-4">
+              <div className="d-flex flex-wrap">
                 <img
-                  className="rounded"
+                  className="rounded mr-2"
                   src="https://upload.wikimedia.org/wikipedia/commons/thumb/b/b6/Image_created_with_a_mobile_phone.png/1280px-Image_created_with_a_mobile_phone.png"
                   style={{ maxHeight: "150px", maxWidth: "150px" }}
                   alt="product"
                 />
-                <div className="align-self-center ml-2">
-                  <h3>Product:</h3>
+                <div className="align-self-center">
+                  <h3 className="mb-0">Product:</h3>
                   <label className="text-capitalize">
                     {order.items[0].product.name}
                   </label>
                 </div>
               </div>
               <Row>
-                <Col>
+                <Col xl="4" xs="6">
                   <div className="d-sm-flex flex-wrap">
                     <h3 className="mb-0 mr-2">Date: </h3>
                     <label>
-                      {buyer.firstName} {buyer.lastName}
+                      {dateTime.getMonth() + 1}/{dateTime.getDate()}/
+                      {dateTime.getFullYear()}
                     </label>
                   </div>
                   <div className="d-sm-flex flex-wrap">
                     <h3 className="mb-0 mr-2">Time: </h3>
-                    <label>{buyer.email}</label>
-                  </div>
-                  <div className="d-sm-flex flex-wrap">
-                    <h3 className="mb-0 mr-2">Total: </h3>
-                    <label>{order.orderAmount * order.quantity}</label>
+                    <label>
+                      {dateTime.getHours()}:{dateTime.getMinutes()}:
+                      {dateTime.getSeconds()}
```

Comment:
Nice! We can go a step further. Try exploring the docs. _hint: each one just uses a method from the Date class_

You could also put these in helper functions to tidy it out a bit.